test/cmd/sdn.sh

```diff
@@ -4,29 +4,82 @@ trap os::test::junit::reconcile_output EXIT
-os::cmd::expect_success 'oc delete namespace sdn-test'
-os::cmd::try_until_failure 'oc get netnamespace sdn-test'
+
+os::cmd::expect_failure 'oc get netnamespace sdn-test-1'
+os::cmd::expect_success 'oc new-project sdn-test-1'
+os::cmd::expect_success 'oc get namespace sdn-test-1'
+os::cmd::expect_failure 'oc get netnamespace sdn-test-2'
+os::cmd::expect_success 'oc new-project sdn-test-2'
+os::cmd::expect_success 'oc get namespace sdn-test-2'
+os::cmd::expect_failure 'oc get netnamespace sdn-test-3'
+os::cmd::expect_success 'oc new-project sdn-test-3'
+os::cmd::expect_success 'oc get namespace sdn-test-3'
+
+os::cmd::try_until_success 'oc get netnamespace sdn-test-1'
+os::cmd::try_until_success 'oc get netnamespace sdn-test-2'
+os::cmd::try_until_success 'oc get netnamespace sdn-test-3'
+orig_vnid1="$(oc get netnamespace sdn-test-1 -o jsonpath='{.netid}')"
+orig_vnid2="$(oc get netnamespace sdn-test-2 -o jsonpath='{.netid}')"
+orig_vnid3="$(oc get netnamespace sdn-test-3 -o jsonpath='{.netid}')"
+os::cmd::expect_success "test '$orig_vnid1' != 0"
+os::cmd::expect_success "test '$orig_vnid2' != 0"
+os::cmd::expect_success "test '$orig_vnid3' != 0"
+os::cmd::expect_success "test '$orig_vnid1' != '$orig_vnid2'"
+os::cmd::expect_success "test '$orig_vnid1' != '$orig_vnid3'"
+os::cmd::expect_success "test '$orig_vnid2' != '$orig_vnid3'"
+
+os::cmd::expect_success 'oadm pod-network join-projects --to=sdn-test-1 sdn-test-2'
+new_vnid1="$(oc get netnamespace sdn-test-1 -o jsonpath='{.netid}')"
+new_vnid2="$(oc get netnamespace sdn-test-2 -o jsonpath='{.netid}')"
+new_vnid3="$(oc get netnamespace sdn-test-3 -o jsonpath='{.netid}')"
+os::cmd::expect_success "test '$new_vnid1' = '$orig_vnid1'"
+os::cmd::expect_success "test '$new_vnid2' = '$orig_vnid1'"
+os::cmd::expect_success "test '$new_vnid3' = '$orig_vnid3'"
+
+os::cmd::expect_success 'oadm pod-network make-projects-global sdn-test-1'
+new_vnid1="$(oc get netnamespace sdn-test-1 -o jsonpath='{.netid}')"
+new_vnid2="$(oc get netnamespace sdn-test-2 -o jsonpath='{.netid}')"
+new_vnid3="$(oc get netnamespace sdn-test-3 -o jsonpath='{.netid}')"
+os::cmd::expect_success "test '$new_vnid1' = 0"
+os::cmd::expect_success "test '$new_vnid2' = '$orig_vnid1'"
+os::cmd::expect_success "test '$new_vnid3' = '$orig_vnid3'"
+
+os::cmd::expect_success 'oadm pod-network isolate-projects sdn-test-1'
+new_vnid1="$(oc get netnamespace sdn-test-1 -o jsonpath='{.netid}')"
+new_vnid2="$(oc get netnamespace sdn-test-2 -o jsonpath='{.netid}')"
+new_vnid3="$(oc get netnamespace sdn-test-3 -o jsonpath='{.netid}')"
+os::cmd::expect_success "test '$new_vnid1' != 0"
+os::cmd::expect_success "test '$new_vnid1' != '$new_vnid2'"
+os::cmd::expect_success "test '$new_vnid1' != '$new_vnid3'"
+os::cmd::expect_success "test '$new_vnid2' = '$orig_vnid1'"
+os::cmd::expect_success "test '$new_vnid3' = '$orig_vnid3'"
+
+os::cmd::expect_success 'oc delete namespace sdn-test-1'
```

Comment:
These should be in a `cleanup` block at the top along with any other cluster resources you create